### PR TITLE
Add notificationUrl to deposit request

### PR DESCRIPTION
**Pull request by colinbird**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by jvc9109 — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

File: openapi/components/schemas/DepositRequest.yaml [hunk 1]
```diff
@@ -125,6 +125,21 @@ properties:
...
+    description: |-
+      URL where a server-to-server POST notification is sent.
+      This notification is sent when the transaction result is finalized after a timeout or an offsite interaction.
+
+      Do not trust this notification alone,
+      complete a GET request to confirm the result of the transaction.
+      To ensure the request is not reattempted,
+      when the result is confirmed, respond with a 2xx HTTP status code.
+
+      The following placeholders are available to use in this URI: `{id}` and `{result}`.
```
Comment (noon-dawg) [inline openapi/components/schemas/DepositRequest.yaml:138]: suggestion: we should use the same text if possible

'Do not trust this notification alone' is an ambiguous statement.

```suggestion
      Do not interpret this notification as a confirmation,
      complete a `GET` request to confirm the result of the transaction.
      To ensure the request is not reattempted,
      when the result is confirmed, respond with a `2xx` HTTP status code.

      The following placeholders are available to use in this URI: `{id}` and `{result}`.
      These placeholders are replaced the with the transaction ID and result accordingly.
```

Comment (ashkarpetin) [reply openapi/components/schemas/DepositRequest.yaml:138]: Hi @noon-dawg this description was taken from Tranasaction.notification

Comment (noon-dawg) [reply openapi/components/schemas/DepositRequest.yaml:138]: Hi @ashkarpetin, thanks for spotting this 🙏 
I've updated the the description in Tranasaction.notication in this [PR](https://github.com/Rebilly/api-definitions/pull/1764)

Comment (ashkarpetin) [reply openapi/components/schemas/DepositRequest.yaml:138]: I am going to merge this, can you please update this description in that PR too

File: openapi/components/requestBodies/PostDepositRequest.yaml [hunk 2]
```diff
@@ -125,3 +125,19 @@ properties:
...
+      URL where a server-to-server `POST` notification is sent.
+      This notification is sent when the transaction result is finalized after a timeout or an offsite interaction.
+
+      Do not interpret this notification as a confirmation,
+      complete a `GET` request to confirm the result of the transaction.
+      To ensure the request is not reattempted,
+      when the result is confirmed, respond with a `2xx` HTTP status code.
+
+      The following placeholders are available to use in this URI: `{id}` and `{result}`.
+      These placeholders are replaced the with the transaction ID and result accordingly.
```
Comment (adamaltman) [inline openapi/components/requestBodies/PostDepositRequest.yaml:139]: what about also `{depositRequestId}`?

File: openapi/components/requestBodies/PostDepositRequest.yaml [hunk 3]
```diff
@@ -125,3 +125,19 @@ properties:
...
+    type:
+      - 'string'
+      - 'null'
+    format: uri
```
Comment (adamaltman) [inline openapi/components/requestBodies/PostDepositRequest.yaml:143]: I think format is `iri` because `{` is disallowed from `uri`.

**Pull request merged**